Update community resources for migration guides (#11376)
withastro/docs: src/content/docs/en/guides/cms/ghost.mdx

## Community Resources 

<CardGrid>

  <LinkCard title="Ghost CMS & Astro Tutorial" href="https://matthiesen.xyz/blog/astro-ghostcms" />
  <LinkCard title="Astro + Ghost + Tailwind CSS" href="https://andr.ec/posts/astro-ghost-blog/" />
  <LinkCard title="Building a Corporate site with Astro and Ghost" href="https://artabric.com/post/building-a-corporate-site-with-astro-and-ghost/" />
  <LinkCard title="`astro-starter-ghost`" href="https://github.com/PhilDL/astro-starter-ghost" />

</CardGrid>

:::note[Have a resource to share?]

> If you found (or made!) a helpful video or blog post about using Ghost with Astro, edit this page and add it here!

Author: ```suggestion
If you found (or made!) a helpful video or blog post about using Ghost with Astro, [add it to this list](https://github.com/withastro/docs/edit/main/src/content/docs/en/guides/cms/ghost.mdx)!
```

## Community Resources 

<CardGrid>

  <LinkCard title="Ghost CMS & Astro Tutorial" href="https://matthiesen.xyz/blog/astro-ghostcms" />
  <LinkCard title="Astro + Ghost + Tailwind CSS" href="https://andr.ec/posts/astro-ghost-blog/" />

>   <LinkCard title="Building a Corporate site with Astro and Ghost" href="https://artabric.com/post/building-a-corporate-site-with-astro-and-ghost/" />
>   <LinkCard title="`astro-starter-ghost`" href="https://github.com/PhilDL/astro-starter-ghost" />
>   <LinkCard title="`astro-ghostcms`" href="https://github.com/MatthiesenXYZ/astro-ghostcms" />

Reviewer: I think we can remove the `astro-ghostcms` link because the repo is no longer maintained:
>  This repository was archived by the owner on Jan 9, 2025. It is now read-only. 
```suggestion
  <LinkCard title="Building a Corporate site with Astro and Ghost" href="https://artabric.com/post/building-a-corporate-site-with-astro-and-ghost/" />
  <LinkCard title="`astro-starter-ghost`" href="https://github.com/PhilDL/astro-starter-ghost" />
```
Or do you think it might still be relevant as an example (this is not too old: `"astro": ">=4.4.1"`)? If so, we should move the "Build a Corporate site..." link at the end for a better balance:
```suggestion
  <LinkCard title="`astro-starter-ghost`" href="https://github.com/PhilDL/astro-starter-ghost" />
  <LinkCard title="`astro-ghostcms`" href="https://github.com/MatthiesenXYZ/astro-ghostcms" />
  <LinkCard title="Building a Corporate site with Astro and Ghost" href="https://artabric.com/post/building-a-corporate-site-with-astro-and-ghost/" />
```
